images/macos/helpers/Common.Helpers.psm1

```diff
@@ -147,4 +146,16 @@ function Start-DownloadWithRetry
     }
 
     return $filePath
-}
\ No newline at end of file
+}
+
+function Add-EnvironmentVariable {
+    param
+    (
+        [Parameter(Mandatory)] [string] $Name,
+        [Parameter(Mandatory)] [string] $Value,
+        [string] $FilePath = "${env:HOME}/.bashrc"
+    )
+
+    $envVar = "export {0}={1}" -f $name, $value
+    Add-Content -Path $filePath -Value $envVar
```

Comment:
nitpick, for consistency with the other functions use capital leters for params' variables:
```suggestion
    $envVar = "export {0}={1}" -f $Name, $Value
    Add-Content -Path $FilePath -Value $envVar
```